readme.md

```diff
@@ -1,7 +1,7 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://kate-prakofyeva.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://kate-prakofyeva.github.io/layout_moyo-header/report/html_report/)
 
 > Follow [these instructions](https://mate-academy.github.io/layout_task-guideline/#how-to-solve-the-layout-tasks-on-github)
```

Comment:
For the future: 
copy DEMO LINK and TEST REPORT LINK to the PR description
point [19.](https://mate-academy.github.io/layout_task-guideline/#create-a-pull-request-pr)